docs/release/trg-8/trg-8-05.md

```diff
@@ -0,0 +1,72 @@
+---
+title: TRG 8.05 - Trivy
+---
+
+| Status | Created     | Post-History                         |
+|--------|-------------|--------------------------------------|
+| Active | 26-Feb-2024 | Initial release                      |
+
+## Why
+
+**Trivy** identifies known vulnerabilities in the packages and libraries within your container images.
+
+:::info
+
+For any errors, please contact Security Team by creating an issue on GitHub.
+
+:::
+
+## Description
+
+Trivy stands as our container vulnerability scanner of choice, ensuring the security of our container images by targeting both OS-level and library dependencies. Here's a concise breakdown of the Trivy integration in our workflow:
+
+:::info
+
+The Trivy scan is initiated either on-demand through manual dispatch or based on a CRON schedule, executing once daily. The job is executed on the latest Ubuntu and requires specified permissions: reading actions and content and writing security events.
+
+:::
+
+The primary step involves the Trivy vulnerability scanner pulling the container image tractusx/irs-api:latest from Docker Hub. Before scanning, it's essential to ensure that the desired image on Docker Hub is correctly configured for the scan.
+
+:::caution
+
+We recommend always scanning the most recently published image to maintain updated security assessments. Utilizing the aquasecurity/trivy-action@0.12.0, the scanner inspects the image for vulnerabilities of types os and library. Results are formatted as SARIF and stored in trivy-results.sarif.
+
+:::
+
+After the scan, results are then uploaded to the GitHub Security tab via the github/codeql-action/upload-sarif@v3 action, ensuring engineers can efficiently review and address any highlighted vulnerabilities.
+
+```md
+/********************************************************************************
+name: "Run Trivy scan and upload SARIF"
+
+on:

```

Comment:
Here Trivy now is not executed on a push to main in contrast to CodeQL and KICS. Seems inconsistent or maybe if it isn't I think an explanation for these differences would be great.